src/en/reference-numbering.md

```diff
@@ -2,8 +2,8 @@ Title: Machine and unit numbering
 
 # Machine and Unit numbering
 
-Within an environment juju keeps track of sequences for machines and units.
-(Prior to 1.25 just machines). This means that for the life of an environment 
+Within an environment Juju keeps track of sequences for machines and units.
```

Comment:
Yeah, I was unsure. In some places this uses enivrionment instead of model, but some places it is less definitely talking about a Juju entity and more about a thing. I will change it here though